src/js/components/Drop/README.md

```diff
@@ -352,6 +352,17 @@ Defaults to
 undefined
 ```
 
+**global.drop.intelligentMargin**
+
+Whether the margin should adjust according to the 
+    drop's alignment with respect to the target. Expects `boolean`.
```

Comment:
I don't think this description is clear enough to understand the goal of this new prop. Elaborating more might help.